setup.cfg

```diff
@@ -99,7 +99,6 @@ asdf_schema_tests_enabled = true
 xfail_strict = true
 qt_no_exception_capture = 1
 filterwarnings =
-    error
```

Comment:
please don't remove it

Reply:
oh sorry, accidentally commited

Reply:
Btw. this was the warning I got:
```
pytest.PytestDeprecationWarning: direct construction of DocTestModulePlus has been deprecated, please use DocTestModulePlus.from_parent
```

Reply:
do you see this locally? If yes, then downgrade pytest version to <5.4

Reply:
yes